functions/units/index.js

```diff
@@ -0,0 +1,49 @@
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+const dbref = admin.firebase().ref();
+
+exports.registerUnit = functions.https.onRequest((req, res) => {
+    if(req.method !== "POST") {
+        return res.status(400).json({
+            message: "Bad request, POST Required"
+        })
+    } else {
+        const data = req.body;
+        dbref.child('Users').child(data.uid).child('Units').push({
+            unit_type : data.unit_type.toString(),
+            fcm_token : data.fcm_token.toString()
+        }).then((unit) => {
+            return res.status(200).json({
+                message: 'Device registered successfully',
+                uid: unit.key
+            })
+        }).catch((error) => {
+            res.json({
+                message: "error" + error.toString()
+            })
+        })
+    }
+});
+
+exports.updateFcmToken = functions.https.onRequest((req, res) => {
+    if(req.method !== "POST") {
+        return res.status(400).json({
+            message: "Bad request, POST Required"
+        })
+    } else {
```

Comment:
You're exiting the function here, no need for else.

Reply:
Got it